Modules/Sources/DesignSystem/Components/FAB.swift

```diff
@@ -2,25 +2,34 @@ import SwiftUI
 
 public struct FAB: View {
     let image: Image
-    let action: () -> Void
+    let action: (() -> Void)?
```

Comment:
It feels pretty strange to have an optional action on a "floating _action_ button" type... Also, I didn't see a place that actually use a nil action (there is one `FAB(action:{})` though). What do you think reverting it back to non-optional?

Reply:
See https://github.com/wordpress-mobile/WordPress-iOS/pull/23454/files#r1700425910. I'm not entirely happy with this API design, but I don't think it's worth changing. 

We've been discussing removing FABs soon anyway 👀